onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -394,89 +394,135 @@ TensorrtExecutionProvider::TensorrtExecutionProvider(const TensorrtExecutionProv
   }
 
   // Get environment variables
-  const std::string max_partition_iterations_env = onnxruntime::GetEnvironmentVar(tensorrt_env_vars::kMaxPartitionIterations);
-  if (!max_partition_iterations_env.empty()) {
-    max_partition_iterations_ = std::stoi(max_partition_iterations_env);
-  }
-
-  const std::string min_subgraph_size_env = onnxruntime::GetEnvironmentVar(tensorrt_env_vars::kMinSubgraphSize);
-  if (!min_subgraph_size_env.empty()) {
-    min_subgraph_size_ = std::stoi(min_subgraph_size_env);
-  }
-
   if (info.has_trt_options) {
```

Comment:
I highly recommend adding a VERBOSE Log line at the end of this constructor which prints out the values of ALL the TRT EP option values. This way we have a way to debug in case user didn't set options correctly. (or mixed env var and options struct)